sql/moz-fx-data-shared-prod/monitoring_derived/topsites_click_rate_live_v1/init.sql

```diff
@@ -11,5 +11,8 @@ FROM
   `moz-fx-data-shared-prod.contextual_services_live.topsites_click_v1`
 WHERE
   DATE(submission_timestamp) > '2010-01-01'
+AND
+  -- filter out the non-adM advertisers: Yandex
+  advertiser NOT IN ('O=45:A', 'yandex')
```

Comment:
I think I'd prefer that we move this filter into the `remotesettings` calculation specifically:

```
COUNTIF(reporting_url IS NULL AND advertiser NOT IN ('O=45:A', 'yandex')) AS n_remotesettings,
```

So that `n` still reflects overall volume of pings. Is it true that `COUNT(reporting_url) AS n_contile` should be unaffected by this filter?

Reply:
> I think I'd prefer that we move this filter into the `remotesettings` calculation specifically:
> 
> ```
> COUNTIF(reporting_url IS NULL AND advertiser NOT IN ('O=45:A', 'yandex')) AS n_remotesettings,
> ```
> 
> So that `n` still reflects overall volume of pings.

Good point.

> Is it true that `COUNT(reporting_url) AS n_contile` should be unaffected by this filter?

Correct.